src/main/java/com/atherys/rpg/service/AttributeService.java

```diff
@@ -129,6 +129,8 @@ public AttributeService() {
      */
     public Map<AttributeType, Double> mergeAttributes(Map<AttributeType, Double> source, Map<AttributeType, Double> additional) {
         additional.forEach((type, value) -> source.merge(type, value, Double::sum));
+        //Added Check for Stats under 0; Replace with 0;
+        additional.forEach((k,v) -> { if (v <= 0){additional.replace(k,v,0.0); }});
```

Comment:
```additional``` is returned but probably not used, this function modifies the ```source``` map, we really should be returning source here I think?

The check most likely needs to be done in the first forEach loop that modifies the ```source``` map